monkey/common/utils/exceptions.py

```diff
@@ -6,7 +6,11 @@ class FailedExploitationError(Exception):
     """ Raise when exploiter fails instead of returning False """
 
 
-class InvalidRegistrationCredentialsError(Exception):
+class CredentialsError(Exception):
+    """ Raise when credentials are wrong"""
+
+
```

Comment:
This is never used, except for `InvalidRegistrationCredentialsError` inherits it.